crates/lair_keystore_api/Cargo.toml

```diff
@@ -28,6 +28,8 @@ tokio = { version = "1.7.1", features = [ "full" ] }
 toml = "0.5"
 rand = "0.7"
 crypto_box = "0.5"
+serde = { version = "1.0", features = ["derive"] }
+serde_yaml = "0.8"
```

Comment:
Should be able to remove these changes, and all the serde additions to the keypair types as well, now.

```suggestion
```